crazyRichAsians/CareRecord
# feat: reset password

Issue: #56

### Objective:
1. Create reset password screen
2. Integrate firebase reset password method

### Solution
![image](https://user-images.githubusercontent.com/18695269/83021598-d1789f00-a05c-11ea-9a46-93bdcc166f5b.png)

### Testing Instructions:
1. Go to forget password screen
2. Enter a email that has been registerd
3. press submit, then check your emails for the reset link
